lib/cisco_node_utils/cmd_ref/bgp_af.yaml

```diff
@@ -74,13 +74,14 @@ dampen_igp_metric:
 
 dampening:
   auto_default: false
+  multiple: true
```

Comment:
It's an error; thought 'multiple' meant 'returns a list', but that is not quite correct.